dhall/src/Dhall/Eval.hs

```diff
@@ -689,6 +689,8 @@ eval !env t0 =
             vCombineTypes (eval env t) (eval env u)
         Prefer t u ->
             vPrefer env (eval env t) (eval env u)
+        RecordCompletion t u ->
+            eval env (Annot (Prefer (Field t "default") u) (Field t "Type"))
```

Comment:
Given that we know that `eval` ignores the type annotation, can we just skip adding it here?
```suggestion
            eval env (Prefer (Field t "default") u)
```

Reply:
In this case I prefer to pedantically follow the standard given that we don't have any benchmarks showing that this is a measurable optimization